client/src/componets/NavBar.jsx

```diff
@@ -58,12 +58,19 @@ const Navbar = () => {
                                 <i class="0bi bi-cart4 nav-link mb-3"> My Cart</i>
                             </Link>
                         </li>
+<<<<<<< HEAD
```

Comment:
this happens when you pull the main and don't see the changes that pull
Take care with this errors